python/pyspark/pandas/internal.py

```diff
@@ -658,7 +654,7 @@ def __init__(
                     data_fields = [
                         field.copy(
                             name=name_like_string(struct_field.name),
-                            nullable=(force_nullable or field.nullable),
+                            nullable=field.nullable,
```

Comment:
We can remove this? The new `nullable` will be reused from the original `field`.